posts/2022-01-18-beta-release-post.adoc

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: "Support for new changes to SSL certificates in Open Liberty 22.0.0.2-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Support for new changes to SSL certificates in Open Liberty 22.0.0.2-beta - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+blog_description: Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Support for new changes to SSL certificates in Open Liberty 22.0.0.2-beta
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.2-beta offers just one new feature, supporting new changes in SSL certificates with urlencoded strings. 
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try either package, <<feedback, let us know what you think>>.
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features:
```

Comment:
Not sure this formatting is correct but there is only one beta package now so...
```suggestion
[#allbeta]
== All Beta Features package
The All Beta features package for link:{url-about}[Open Liberty] contains all Open Liberty beta features and GA features and functions.

This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.

If you try the beta package, <<feedback, let us know what you think>>.

The following beta features are available in 22.0.0.2-beta:
```